webapp/graphite/render/views.py

```diff
@@ -572,3 +581,8 @@ def errorPage(message):
   template = loader.get_template('500.html')
   context = Context(dict(message=message))
   return HttpResponseServerError( template.render(context) )
+
+def log_expensive_request(start, request):
```

Comment:
does slow imply expensive? i'd prefer renaming this to `log_slow_request` or something. slow implies time, expensive can imply time, cpu, memory, network, money, etc.